metricbeat/module/vsphere/_meta/docs.asciidoc

```diff
@@ -0,0 +1,4 @@
+== vsphere Module
+
+This is the vsphere Module.
```

Comment:
Could you add some more details here, for example link to vSphere (installation) page or docs? Also did you test with a specific version?